docs/README.md

```diff
@@ -480,6 +491,64 @@ We've seen:
 ### Alter column
 ### Create index
 ### Create table
+
+A create table migration creates a new table in the database.
+
+**create table** migrations have this structure:
+
+```go
+type OpCreateTable struct {
+	Name    string   `json:"name"`
+	Columns []Column `json:"columns"`
+}
+
+```
```

Comment:
I would rather go for JSON based docs rather than Go. Many users won't know Go and that should be fine?

Reply:
* d4261d843522b27aa5ff6270108b8419b8a3d7f2 uses an ad-hoc JSON-style description of the operation.
* e405a8075b72773922b580506d661c1eda91bcaa uses Typescript descriptions.

Personally I prefer the latter, but happy to go with the former. Using [JSON Schema](https://json-schema.org/) would be another option but it seems excessively verbose to be useful as an informal description; it feels more geared towards schema verification.

Reply:
I like the JSON-style more because it's language agnostic, users don't really need to know any programming language, but they need to understand JSON. Approving the PR so we don't block docs, I would go for https://github.com/xataio/pgroll/commit/d4261d843522b27aa5ff6270108b8419b8a3d7f2 and iterate from there